.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -80,6 +82,18 @@ jobs:
           ACR_REGISTRY_RESOURCE_NAME: ccfmsrc
           CCF_AZURE_SUBSCRIPTION_ID: $(CCF_AZURE_SUBSCRIPTION_ID)
 
+      - script: |
+          set -ex
+          set -o pipefail
+          mapfile -t IP_ADDR_LIST <<< $(cat ~/ipAddresses | awk '{print $2}')
+          ssh agent@${IP_ADDR_LIST[0]} -o "StrictHostKeyChecking=no" -o ConnectTimeout=100 '
+          nproc --all
+          lscpu
+          cat /proc/cpuinfo 
+          '
+        name: print_cpu_stats
+        displayName: "Print CPU stats"
```

Comment:
Dumping some extra info. Not strictly necessary, but maybe it'll magically be useful in future.